src/Model/Scope/Scope.php

```diff
@@ -322,7 +317,7 @@ public function or(AbstractScope $scope)
      */
     public static function mergeAnd(AbstractScope $scopeA, AbstractScope $scopeB, $_ = null)
```

Comment:
what is the use case?

I think is it when wrong - and self::AND will be used as a value instead of operator

exactly a reason why to not allow these flexi constructors